### PR TITLE
Bug/jkw countdown timer context

### LINKED ISSUE #59: Fix Timer component bug with incorrect time on page load
### Feature Categories:
- React
- Bug

**Describe the feature/Bug your gonna work on**
Currently, there is a bug with the <Timer> component from React Compound Timer, where its state does not update on first page load with the settings Context values. The CompoundTimer component (built by us) does receive the updated value, but it's not making its way into Timer state. 

**Additional context**
We tried making use of the built-in setTime method, and did not have luck with that yet, but we can revisit that.

### PR DESCRIPTION
**Pull request by jkwest-93**
# Som Timer Pull Request Form
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- After connecting our `CountdownTimer` to the backend & context, we realized there was a bug with getting the time to re-render automatically. This PR fixes this bug, so that the information that we `GET` from the database timers automatically loads into our timer functionality
     - For now, this removes our color functionality (though we think there's a way to get this back using some keyframe animations!)
     - We also added in a conditional render that toggles the `Play`/`Pause` buttons & appropriate styling; this allowed us to add the `Skip` button.
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
Run `npm test` in the terminal
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

- Closes #59

### CODE REVIEW

File: src/Components/CountdownTimer/CountdownTimer.js [hunk 1]
```diff
@@ -35,23 +35,15 @@ const CountdownTimer = () => {
...
-          {
-            time: settings.workInterval * 15000,
-            callback: () => countdownColorChange("#e44f47"),
-          },
         ]}
       >
-        {({ start, pause, reset, getTime}) => (
+        {({ start, pause, getTime, getTimerState, setTime }) => (
           <>
+            {
```
Comment (rwilliams659) [inline src/Components/CountdownTimer/CountdownTimer.js:42]: Y'all are geniuses for thinking of this solution!  Amazing work!

**Review by rwilliams659 — APPROVED** [review]
So impressed with your mad problem-solving bug-fixing skills. Way to crush it!

File: src/Components/CountdownTimer/CountdownTimer.js (hunk 1 above)
Comment (Abdeboskey) [reply src/Components/CountdownTimer/CountdownTimer.js:42]: I'll miss that particular animation, but I'm so glad we got this fix!

File: src/Components/CountdownTimer/CountdownTimer.js [hunk 2]
```diff
@@ -92,36 +84,54 @@ const CountdownTimer = () => {
...
+                  aria-label="pause"
+                  onClick={pause}
+                >
+                  <img
+                    className={style.pauseTimerControlIcon}
+                    src={pauseTimerIcon}
+                    alt="pause symbol"
+                  />
+                </button>
+              }
```
Comment (Abdeboskey) [inline src/Components/CountdownTimer/CountdownTimer.js:112]: The way that these buttons render is really satisfying

File: src/Context/SettingsContext.js [hunk 3]
```diff
@@ -4,26 +4,26 @@ import { getSettings } from '../apiCalls'
...
-        <SettingsContext.Provider value={[ settings, setSettings ]}>
-            {children}
-        </SettingsContext.Provider>
-    )
+
+  return (
+    <SettingsContext.Provider value={[ settings, setSettings ]}>
+      {children}
+    </SettingsContext.Provider>
+  )
```
Comment (Abdeboskey) [inline src/Context/SettingsContext.js:28]: Not sure why we are getting this weird little bug where the settings are being reset to our default periodically?

**Review by Abdeboskey — APPROVED** [review]

File: src/Components/CountdownTimer/CountdownTimer.js (hunk 1 above)
Comment (sienna-kopf) [reply src/Components/CountdownTimer/CountdownTimer.js:42]: So glad you got this bug figured out!

**Review by sienna-kopf — APPROVED** [review]

**Pull request merged**